src/controllers/user/collection/routes/index.js

```diff
@@ -17,14 +17,14 @@ module.exports = function (trackService) {
     try {
       const [countResult] = await sequelize.query(`
         SELECT count(*) as count FROM(
-          SELECT track.tid
+          SELECT track.id
           FROM tracks as track
-          INNER JOIN rsntr_usermeta AS um ON (track.uid = um.user_id AND um.meta_key = 'nickname')
-          INNER JOIN plays AS play ON (play.tid = track.tid AND play.event = 1 AND play.uid = :listenerId)
+          INNER JOIN rsntr_usermeta AS um ON (track.creator_id = um.user_id AND um.meta_key = 'nickname')
```

Comment:
Not for this PR but just a mental note that we should remove this table and all the dependencies on it.